Resfeber/Views/WaterfallLayout.swift

```diff
@@ -9,93 +9,93 @@
 import UIKit
 
 protocol WaterfallLayoutDelegate: AnyObject {
-    
     func collectionView(
         _ collectionView: UICollectionView,
-        heightForPhotoAtIndexPath indexPath: IndexPath) -> CGFloat
+        heightForPhotoAtIndexPath indexPath: IndexPath
+    ) -> CGFloat
```

Comment:
This format looks very strange. Consider 1 or 2-line alternatives:
```
func collectionView(_ collectionView: UICollectionView, heightForPhotoAtIndexPath indexPath: IndexPath) -> CGFloat
```
or
```
func collectionView(_ collectionView: UICollectionView, 
                    heightForPhotoAtIndexPath indexPath: IndexPath) -> CGFloat
```

Reply:
Fixed